extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -308,6 +308,11 @@ public function getWikiVariables() {
 			$wikiVariables['facebookAppId'] = $egFacebookAppId;
 		}
 
+		$wikiImages = (new WikiService())->getWikiImages( [$this->wg->CityId], 500 );
+		if ( is_array( $wikiImages ) && is_string( $wikiImages[$this->wg->CityId] ) ) {
```

Comment:
@mklucsarits -- PHP will definitely complain if you try to access a non-array as an array or a key of an array that doesn't exist, but not within ```empty```.  I think its worth using empty here; applying this level of scrutiny may cause confusion for future coders about what is so special about this return value.